python/pyspark/sql/functions.py

```diff
@@ -1258,7 +1302,7 @@ def hash(*cols):
                'uppercase. Words are delimited by whitespace.',
     'lower': 'Converts a string column to lower case.',
     'upper': 'Converts a string column to upper case.',
-    'reverse': 'Reverses the string column and returns it as a new string column.',
+    'reverse': 'Reverses a string column and returns it as a new string column.',
```

Comment:
ditto. I guess we should revert this change or match this change to R/Scala ones too if it matters.